doozerlib/cli/release_gen_payload.py

```diff
@@ -170,16 +189,32 @@ def _get_payload_images(self, images):
 
         return payload_images, invalid_name_items
 
-    def _get_latest_builds(self, payload_images):
+    def _get_latest_builds(self, payload_images: List[ImageMetadata]) -> Tuple[List[BuildRecord], List[ImageMetadata]]:
         """
-        find latest brew build (at event, if given) of each payload image.
+        Find latest brew build (at event, if given) of each payload image.
+
+        If assemblies are disabled, it will return the latest tagged brew build with the candidate brew tag.
+        If assemblies are enabled, it will return the latest tagged brew build in the given assembly. If no such build, it will fall back to the latest build in "stream" assembly.
```

Comment:
If `assembly.stream` does not exist, we should fall back to `builds[0]`. Right now, vertical pod autoscaler is failing to build (https://github.com/openshift/vertical-pod-autoscaler-operator/pull/91). Seems reasonable to fall back to the actual latest build as a last resort.

Reply:
@jupierce `builds[0]` could be in a different assembly which is not `assembly.<runtime.assembly>` or `assembly.stream`. I think we'd better to default to the latest build without `.assembly.` in its release?

Reply:
That's a good idea. Should never happen in our flows, but makes sense for completeness.